service/src/main/java/org/cbioportal/genome_nexus/service/GenomicLocationAnnotationService.java

```diff
@@ -39,6 +39,7 @@
 
 import java.util.List;
 import java.util.Map;
+import java.util.function.Function;
```

Comment:
Maybe we don't need to import this? And same for other files below

Reply:
@leexgh Xiang, thanks! I'm going to optimise imports.